### DISCUSSION TITLE
Clarifying on Legalities of Code Ownership

### DISCUSSION THREAD

**Discussion by shadowtime2000** [alephjs/aleph.js#149] [General]
The legalities of code ownership aren't as transparent right now, and I think it is worth it clearing it up when the Aleph is in earlier stages like right now. In some places it says the owner/authors are the aleph.js authors, while in some places it says it is owned by @ije's company postUI. The question is, is Aleph owned by a company or is it completely open source?

cc @ije

**Comment by ije** [comment]
it is open source under **MIT** license absolutely, all right is reserved by postUI. like nextjs is owned by vercel but it's open source as well.

**Reply by shadowtime2000** [reply]
Interesting. May be a good idea to add a CLA bot to make sure code is legally owned by postUI and maybe also do some sort of license thing to say it will never become closed source or something to not have the risk of an elastic search like incident.

**Reply by ije** [reply]
i think the CLA signature is a bit trouble for contributing and there is no any reason this project will become closed source, and the deno.land also will keep all the releases(source code)
